integration-libs/epd-visualization/root/services/storage-api/storage-api.service.ts

```diff
@@ -0,0 +1,91 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import {
+  EpdVisualizationConfig,
+  EpdVisualizationInnerConfig,
+  VisualizationApiConfig,
+} from '../../config/epd-visualization-config';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StorageApiService {
+  private baseUrl: string;
+
+  constructor(
+    protected http: HttpClient,
```

Comment:
For communication with backend API, by convention we use layers of: Connector and Adapter. For consistency, would you mind please changing your API services to become connector + adapter? I understand this might seem just a "cosmetic" change only in your case, but it's for consistency. As an example, you can see other features. For more, see docs https://sap.github.io/spartacus-docs/connecting-to-other-systems

Reply:
I have some changes to implement this in this PR:
https://github.com/SAP/spartacus/pull/14600
It should be merged tomorrow.

Reply:
Thanks. FYI, I left one comment in that PR https://github.com/SAP/spartacus/pull/14600/files#r767855087

Reply:
I updated the code so that the services use the connectors rather than adapters. I also added a `core` entry point and moved the global services there.
The changes are now present in this PR.